ui/admin/app/styles/app.scss

```diff
@@ -173,8 +173,6 @@
 
 // Onboarding styles
 .create-resources {
-  display: flex;
```

Comment:
Wondering why we removed `display: flex;`? Merely curious is all!

Reply:
oh yup, this was to fix display issues this was causing (the frame was off centered.